core/src/components/datetime/test/minmax/datetime.e2e.ts

```diff
@@ -47,4 +47,33 @@ test.describe('datetime: minmax', () => {
     expect(nextButton).toBeDisabled();
     expect(prevButton).toBeEnabled();
   });
+
+  test.describe('when the datetime does not have a value', () => {
+    test('all time values should be available for selection', async ({ page }) => {
+      /**
+       * When the datetime does not have an active value, all time values should
+       * be available for selection.
+       */
```

Comment:
~ It may be good to clarify somewhere in here that we're specifically testing when today's date falls outside the min/max.